hydradx/notebooks/Omnipool/LP_returns_analysis.ipynb

```diff
@@ -0,0 +1,392 @@
LTZWXfOOcb5b74RBR4kBx8sXqXLTu6/cWNs7Xv9dfFdWLECWLZMJAKvLk7uPDXUWv4vpPj377/Aiy8KJ2RNIL93K1YYl0uBW4apkVhHEESsyOvfddclth2REK0De9w4fVpeL2XkiUSmZ1ArxUvi6awjiCSGxDqCIOJDZSUaLN6CtBKg01KH7TwevRO8YAFw9tn6umHD4PIbO8AGIW3fPmDRIud25ObarvJWAZkNC4AnnwTee8/6HCrmjv+AAc7njgTpPDHTsGH1j00Q8SRcPjgpXplFrHD7f/llqKNORRHroDnrAqW6k8CVnqWLM+ZzVFYaBalIisI89ph12KS8Tt17r3g1O9VqW6zLyTFWdgWAo48Gzj1XTJvFOvWz+fxz4MgjRb49FauOTnWvRbt3R77tMcc4r7cT68z/9yuuEGKwLOAzfryoVF5cDKxfb398zsX3ZerUyNtsdX7JnXeK14MPFuHSSlVygiCIqJADXIkOe60pZGoZ6RosKTGul/cnq+d6OTBSnVQXZhKdC5AgLCCxjiCI+PDtt/D4ga6LgJxdugDmMfeVVbHOArbVzpIHa+u/2f3Tu7dzOz0e4UZRz2m37bp1xvl4VH80t3fUKGDiRBF+RhDJhJXQpeaik446u9+F2T0nOx7z5gGffWZ/XoOzTlwrSpfp4YaZPQ+yd9ZVVgJnnaXPDx5sf55whMtvlCzFA9S8RpJ33jFuY5dv8/jjQ5eZRT9JpKFXkVQq/PDDyLY1f7ekM9nOTWEVSmUWOFU4F7n6LrrIuR1m7ETg228Xr7m5wjUuq4oTBEFEy/4U0mvF2rVCoOvZU8zbpbWwuhfXRIEJEuuIJITEOoIg4g5Xni9CXGsej2MHzZViTCQbUK9SJ54YapNXE89GInh5vZahSU03AR2XmBZefLGpcTVwyRwyRIwq1sSxCaI6WIWqjhmjT9tV3ZOoztR16+xdpYAxL5xFGKyKOzNHF6bKy0WOSkAsCwSM4etWYlSkhBPr1q6N/djxxFwxt7zcKFgCQP/+1vtaOQrtOizhXHCSSDqYJ58s0hWEc28+/rhxvmNH8Rpt6NMbb1iLq2b3dKRYiXV2KQ4IgiBiYX9/LkxJEc/w8p5tvh84OetqQqyjkFoiCanWVYAxdg1jbD5jbAFj7FptWS/G2O+MsXmMsU8ZY9nK9rcyxpYzxpYwxo5Rlg/Tli1njI2zOBVBEHWAoDDn1Fdzu52ddT5jUvtAOJOGmgRfzWFkh9drKeoVrLMoaGF2klTHpWOHmjCdIJKJ5s1Dl1lVbLNDFUfM+dPMXK/npIPHo//2XDYXAHnsSZOAgw4C7r9ff9D3eETOtM8/j7ytVoRzfc2fX73jxwuzsy7akE4zdmLdb7/Z76MW35GioVUeQBWPx+jia9dOhCSrzJ5tnJchttF2qs47D3j3XTG9ZYuepy+cO9IupNdKZHRIwUAQBBE15oGPq68Wr04DX3URKdaZB0Fk5XXzfQGofhjsWWcBI0cal8n8rwSRRMQs1jHGugO4BMAAAL0AHM8Yaw/gZQDjOOc9AHwI4CZt+64AzgTQDcAwAM8zxtyMMTeAiQCOBdAVwFnatgRB1FWUB4wQZx3n1mXY5fb/zTbOhzM+qIUZwlWvBEQHXB2tHDo0/D6AcAaZ3SqxMn68yPP188/7f5gDUXcZNix0WTSVUTdv1kWNn35y3lbNYff77+K3sXYtmJ2zwCyy3HWX7mzauxdo3x4YPjzytloRzlmXLEhR8cUXgXvuAR55xLg+WndGOLebFap4Jh2ZN9zgvI/bbTzXypXAzTcDd9xh7fgDhHgWCIh8dHYcfLAxF6rkkUeAbdtESHCTJmKZ+j0aP160QfLLL0KcltVyVebMCV2WLFWDCYLYPzBfu486SjxDm6/xdR0p1s2caXQ7Z2YC+fnWAz/Vdda98w7w/vvGPHlK4TmCSBaq46zrAuBPznkJ57wKwI8ARgDoCEA+lX8D4FRt+iQA73DOyznnqwAshxD6BgBYzjlfyTmvAPCOti1BEHUMKaxxzoPTfrM5JRCwruyk4VrrkAzcCjUfkZ0TRi0R7/cbO2iROoXiWVXw5ptFp/OQQ+J3TIKoCcxV6KIRJDp0AK69NrJtzZWXAWDqVHA7B5WVI0o+dMsiA9UlnFhn5TxMBLKi9qRJohjGmjVi/uqrRefjzz+t97Or9huLWKf+P6Q7wup/qiKddevWGd18Dz4IHHEEsGuXXjxDbduECcCMGWK+TRt93ZNPitf27YH77gs939y5oiCHpKzM2DEcN85YnESGV5sdmrt2AW++GXp8WR2ZIAgiHpgHcyMZkK6LyHvFtdcCzzwjppctA15+WRQJsiJeYbDxLFBBEDVAdcS6+QAGM8YaMcbSAQwHUABgAXSxbaS2DABaAChS9l+nLbNbThBEXcJUQbBK6+f6zf1dzp3Fumhztr//vkhY/9xz9tuoydX9fj1k6+KLRfgcQRDWjB8P/PWXKAgxbJhlvkdHzIUOouGee8Bfn2a9zil80U6EipZwYp1dIYbaxs4NcP/9QNeu9vnq7Dp+sbwvVfSKRqyrqgIKCqxdkA8+GBoWtWkTcM01+vzzzwP//CMKO8jzZWYaB2EmTNCn//1Xn542zfp7dOut4lV+Pm+8YVwvxVAzJNYRBBFPzGJdJMV76iLqveKHH8TrzTc771OdMFj1um+Vm5cgkoiYxTrO+SIA4wF8DWAGgNkA/AAuAnAFY2wWgCwAcfsVMMbGMMb+YYz9s2XLlngdliCIeKA6DRgzFJkwkJcX2olTwqXMYa+2x5E0bw507w6MHWu/jfqA4/eLMCtAhN6pzgyCIIx4vcABB4hcLl9+GX1nIZIHYQe3AC+2cdo6iXXx6tCEO06sxQniTaNG1svDuSD37LFeHo2zjnPxp/4/5ABIuOO43bor0CpMescO0RHr0cP+s05LExVXH3hAz3OYmWkUzq680npfn8/6e/TII8Crr+rn3G6qUC6dH4BwSMuQ3UMPtT4PQ
```

Comment:
Where do we have data supporting that we have observed 5% 24hr trade volume recently? I think it's lower. We should remove this claim or have data to support it IMO.

Reply:
Also, can we put legends on these graphs so we can tell which color corresponds to what value?